zqd/search/search.go

```diff
@@ -21,14 +21,15 @@ import (
 	"github.com/brimsec/zq/zng/resolver"
 	"github.com/brimsec/zq/zqd/api"
 	"github.com/brimsec/zq/zqd/space"
-	"github.com/brimsec/zq/zql"
 	"go.uber.org/zap"
 )
 
 // This mtu is pretty small but it keeps the JSON object size below 64kb or so
 // so the recevier can do reasonable, interactive streaming updates.
 const DefaultMTU = 100
 
+const DefaultStatsInterval = time.Millisecond * 500
```

Comment:
```suggestion
const StatsInterval = time.Millisecond * 500
```
Since this is an actual value, not a default value.

Reply:
👍